### ISSUE TITLE
`update` when height is 0 causes problems

### ISSUE THREAD

**Issue by seadowg** [Armen101/AudioRecordView#11]
It seems that if `update` is called while the view's height is 0 (which can happen if the view is `GONE` or before it is properly rendered) it causes it to stop working properly - no matter what amplitude is passed it will just show as a straight bar.

It looks like this is happens because the `chunkMaxHeight` is (incorrectly) recalculated [here](https://github.com/Armen101/AudioRecordView/blob/0645d49d8b47119952c19f10b3ec5d8487551de7/audio/src/main/java/com/visualizer/amplitude/AudioRecordView.kt#L139) as it will be greater than the height (minus padding) and fire the `else if`. When the view is displayed and the height updates to the correct value the `chunkMaxHeight` will not be recalculated, and so we're stuck with the incorrect value.

Our workaround at the moment is to skip calls to `update` when `AudioRecordView#getHeight()` is 0:

```java
if (audioRecordView.getHeight() > 0) {
    audioRecordView.update(amplitude);
}
```

**Comment by Armen101** [comment]
Yes, you are absolutely right, this has already been fixed and will be available in the next version. Thanks for your support!

**Comment by Armen101** [comment]
This should be fixed in version **1.0.5** (implementation 'com.github.Armen101:AudioRecordView:1.0.5')
Please let me knew in case of any problem. Thanks.
